ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -19,8 +14,8 @@ import Box from '../../ui/box';
 import Button from '../../ui/button';
 import InfoTooltip from '../../ui/info-tooltip';
 import Popover from '../../ui/popover';
-import Typography from '../../ui/typography';
 import AppLoadingSpinner from '../app-loading-spinner';
+import { Text } from '../../component-library';
```

Comment:
If you would like to use the `ButtonLink` and `Button` components from the component-library folder update this to
```suggestion
import { Text, Button, ButtonLink } from '../../component-library';
```